### DISCUSSION TITLE
Taking Inputs from Edgeapp work done in 3GPP for EdgeCloud Services

### DISCUSSION THREAD

**Discussion by babunkj** [camaraproject/EdgeCloud#183] [Q&A]
Edge Services have been defined in 3GPP as part of the EdgeApp framework in Release 17. The Operator Platform Telco Edge Requirements defined by GSMA, as given in the Document OPG.02 - Operator Platform Telco Edge Requirements, also refers to the 3GPP EDGEAPP architecture. While Camara is defining the Edge Services primarily based on the Telco Edge Requirements defined by GSMA, how can these service definitions take inputs and leverage from the work done in 3GPP. Specifically for the UNI Interface, for which the GSMA UNI document has endorsed the 3GPP EdgeApp procedures and APIs. These have been specified in "Document OPG.05 - User-Network Interface APIs".
[EdgeServices-3GPP-Camara-v1.pdf](https://github.com/camaraproject/EdgeCloud/files/11616479/EdgeServices-3GPP-Camara-v1.pdf)

**Comment by Kevsy** [comment]
Hi @babunkj  -
"Camara is defining the Edge Services primarily based on the Telco Edge Requirements defined by GSMA," -> also the 5GFF (see the two Edge Discovery APIs), but the goal of CAMARA is to produce 3rd-party-developer-facing APIs that represent intents and are not bound to a specific architecture. For example, there may not be a UNI client on the device calling the API. But note we already take inputs from GSMA OP, EdgeXR and 5GFF APIs  and are working on [harmonising them](https://github.com/camaraproject/EdgeCloud/blob/main/documentation/SupportingDocuments/Harmonisation%20of%20APIs/describing%20and%20harmonising%20the%20Edge%20APIs.md)

**Comment by babunkj** [comment]
Agree that the objective is to arrive at Intent based services and not bound to any technology or architecture, though the realization of the services will have a dependency on what the platforms can support. Since the requirements given in GSMA documents have been based on the 3GPP specifications too, I suppose the contributions from GSMA OP would have accounted for them.
With regard to the need for a UNI Client, is that something that is planned to be discussed as part of harmonisation work?

**Comment by Kevsy** [comment]
> I suppose the contributions from GSMA OP would have accounted for them.

That's correct, but there is no obligation for an operator implementing the Edger Cloud (or other CAMARA API) to have the GSMA OP backend. The important part is that the intent can be met, the API caller should not care how the API call is fulfilled.
> With regard to the need for a UNI Client, is that something that is planned to be discussed as part of harmonisation work?

Yes, that will be discussed.
